generic-database-plugin/docs/Database-action.md

```diff
@@ -14,42 +14,27 @@ For example, you may want to run a sql update command on a database before the p
 
 Properties
 ----------
-**Database Command:** The database command to execute.
-
-**Host:** host where database running on.
+**Driver Name:** Name of the JDBC driver to use.
 
-**Port:** database port.
+**Connection String:** JDBC connection string including database name.
 
-**Database:** database name.
+**Database Command:** Database command to execute.
 
 **Username:** User identity for connecting to the specified database.
 
 **Password:** Password to use to connect to the specified database.
 
-**Driver Name:** Name of the JDBC driver to use.
```

Comment:
why is this removed

Reply:
I was moving the properties so they appear in the same order as on UI but accidentally removed this. Fixed.